src/json-schema.js

```diff
@@ -134,13 +134,74 @@ function checkSchemaHasReferences(schema) {
   });
 }
 
+/** Traverses the entire schema to find all of the references
+ * @returns array of each reference that is found in the schema
+ */
+function findReferences(schema) {
```

Comment:
This method is pretty identical to the above `convertSubSchema` function, instead of converting. It simply returns all of the references found in the schema.